src/azure.js

```diff
@@ -82,3 +82,86 @@ api.declare({
     expiry:   expiry.toJSON()
   });
 });
+
+api.declare({
+  method:     'get',
+  route:      '/azure/:account/blob/:container/:level',
```

Comment:
How about `'/azure/:account/containers/:container/:level',`

The other one we use: `/azure/<account>/table/<table>/<level>`
And we probably should have used `/azure/<account>/tables/<table>/<level>`. So that matches with the route for listing.